build_library/prod_image_util.sh

```diff
@@ -247,3 +247,42 @@ create_prod_tar() {
   sudo losetup --detach "${lodev}"
   upload_image "${container}"
 }
+
+DEFAULT_SYSEXTS=(
+  flatcar-zfs:sys-fs/zfs
```

Comment:
```suggestion
  zfs:sys-fs/zfs
```
How about prepending `flatcar-` in the function below because this is not free form but part of the contract to identify the payloads.

Reply:
I'll do this, but i started to question: why the need for the prefix?

Reply:
The idea is to distinguish Flatcar-provided extensions from user-provided extensions, e.g., in the output of `systemd-sysext status`

Reply:
done